src/codeu/chat/common/UserAccessLevel.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Uuid;
+
+
+/**
+ * Created by Lysander on 7/25/17.
+ */
+public class UserAccessLevel {
+    private Uuid user;
```

Comment:
A small thing: If a field is set in the constructor and can't be changed, I would always make it `final`.

Rather than trying to explain that myself, I suggest finding a copy of _Effective Java_ and reading Item 15, "Minimize Mutability". (While you're at it, I strongly recommend reading the whole book!)